queries.sql

```diff
@@ -11,4 +11,49 @@ SELECT (name, secape_attempts) FROM animals WHERE weight_kg > 10.5;
 
 SELECT * FROM animals WHERE neutered = TRUE;
 
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 and 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 and 17.3;
+
+-- Defining the species
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name = '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species = NULL;
+SELECT * FROM animals;
+COMMIT;
+
+-- deleting and ROLLBACK
+BEGIN;
+DELETE FROM animals;
+SELECT * FROM animals;
+ROLLBACK;
+
+-- DELETE ANIMALS WITH CONDITIONS
+BEGIN;
+DELETE  FROM animals
+WHERE date_of_birth > '2022-01-01';
```

Comment:
- [ ] As per project requirements, after deleting animals born after 2022-01-01, you are expected to create a save point that is missing from your code.
`SAVEPOINT SP1;`
![Screen Shot 2023-08-31 at 5 58 51 AM](https://github.com/najibullahjafari/vet-clininc/assets/62849872/13cce934-15c9-43b2-9a42-c59bceb6b28d)

Reply:
well resolved! ✔️